---
title: "Your Title Here"
author: Adelle Marie Sumabat
date: October 16, 2019
output:
  word_document: default
  pdf_document: default
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

Bacteria is found in high concentrations in built environments. Out of all rooms in a built environment, it has been found that bathrooms contain a higher concentration of bacteria than any other room due to bathrooms being a less ventilated room and a much more moist environment. [@ojima2002hygiene] Research has shown that the highest concentration of bacteria in residential homes are found in moist areas such as bathroom sinks. [@rusin1998reduction] Human occupancy and human contact can have an affect on the community of the microbiome of a surface area. [@ramos2014tools] 

This research is important because we spend most of our time indoors and especially in our homes. Being constantly exposed to the bacteria in our homes or public places can greatly influence our health. By knowing how much bacteria is in a certain place we come into close contact with, such as bathroom or restroom sinks, we can then find ways of reducing any contamination that could lead to possible illness. 

# Methods

## Field Sampling

To obtain the samples I used for this experiment, I used sterile cotton swabs, sterile buffer, sterile gloves, and tubes. I swabbed the drains of bathroom sinks from three different homes as well as restroom sinks from three different public places. For the personal bathrooms, I swabbed the bathroom sink in my home, a bathroom sink in my cousin’s home, and a bathroom sink in a friend’s home. For the public restrooms, I swabbed the restroom sinks at a Starbucks, Apple retail store, and a Burger King. This totaled to six distinct sites. I used a sterile swab and inserted it into the drain of a sink. While the swab was in the drain, I moved it in a circular motion for approximately 10 seconds. I removed the swab from the sink and broke off the tip. I inserted the tip of the cotton swab into a labeled tube and set it aside for later use. I swabbed each sink twice and repeated the swabbing procedure for the sinks at each of the six sites. This allowed me to obtain a total of 12 samples. I labeled the tubes from personal bathrooms as 1H, 2H, and ,3H. The tubes from public restrooms were labeled as 1P, 2P, and 3P. Six samples, one sample from each location, were used for culturing and the other six were set aside and frozen at -20 degrees celsius which were later used for Illumina sequencing. 

## Culture and Dilution Plates

A sample from each location were cultured and diluted. The plates I used for culturing were made up of trypdic soy agar (TSA) medium on 100mm petri dishes. I added 200 uL of sterile buffer, PBS, to one of the tubes containing the swab. I vortexed the tube from approximately 15 seconds. 100 uL of this solution was added to a culture plate and spread with beads. I then added 20 uL of the solution to another tube along with 180 uL of PBS. 100 uL of this solution was added to a culture plate, spread with beads, and labeled as the 10x dilution. I did another dilution by adding 20 uL of the 10x solution to another tube and added another 180 uL of PBS. 100 uL of this solution was added to a culture plate, spread with beads, and labeled as the 100x dilution. I repeated this process with each of the six samples and obtained a total of 18 culture and dilution plates. The plates were incubated overnight at 37 degrees Celsius. The following day, the number of colonies and morphotypes were counted and documented. 

## DNA Extraction 

After the colonies and morphotypes were counted and documented, I chose six different colonies to extract DNA from. I labeled 1.5 mL tubes for each sample. I added cells from one colony to a tube using a sterile pipette tip. I followed the manufacturer’s protocol from the Sigma REDExtract-N-Amp kit

## PCR

For PCR, I had to first make my master mix after calculating the volumes of reagents needed in my mix. My calculations indicated that I needed 77 uL of AMP, 6.2 uL of 27f primer, 6.2 uL of 92r primer, 7.7 uL of BSA, and 49.3 uL of H20. After I created my master mix, I added 19 uL into 7 tubes. 1 uL from each sample was added to a tube. I added 1 uL of H20 to the seventh tube to serve as the negative control.

The samples were put into a PCR thermocycler. The PCR cycle condition began with five minutes of being in 95 degrees celsius. The thermocycler then changed to 94 degrees celsius for 30 seconds, 65 degrees celsius for 30 seconds, and 72 degrees celsius for one minute. This was repeated for three cycles and the temperature was stepped down by 1 degree per cycle. The thermocycler then changed to  94 degrees C for 30 seconds, 55 degrees C for 30 seconds, and 72 degrees C for 1 minute. This was repeated for 25 cycles. Finally, the thermocycler changed to 72 degrees C for 10 minutes and was held at 4 degrees C. 


## Gel Electrophoresis 

The gel used for my experiement contained 2% agarose gel and SYBR safe dye. 4 uL of our DNA samples were loaded onto a 50 well gel. Ladders served as a positive control at each ends of the wells. I ran the gel under 140V and X TAE buffer conducted current through the gel.

## Sanger Sequencing

After checking for successful amplification using gel electrophoresis, PCR products were cleaned with ExoSAP (Invitrogen) and sent for unidirectional Sanger sequencing at MCLAB (South San Francisco, CA) using the 27f primer.


## Data Analysis 

The alignment program I used to analyze the Sanger sequences is called Geneious Prime. I received the results from Sanger sequencing as a zip file. I imported my files to geneious and was able to analyze my sequences. I indicated which sequences failed as well as sequences that failed but could have been used for BLAST. I cleaned up and trimmed the sequences that I thought to be usable for BLAST and used IUPAC ambiguity codes for bases that were hard to identify. 

I used an application called BLAST to match my samples to organisms. 
I also used Geneious to create a Bayesian phylogeny and a Maximum Likelihood phylogeny. Thermus aquaticus was used as an outgroup for both phylogenies. The Bayesian phlyogeny was used 


# Results

Results from the bacteria cultures showed that personal bathrooms contained too many colonies to count. For the purpose of this experiement, I documented that I counted 1000 colonies from the plates that were uncountable. I used  a high value to represent the high number of colonies that were uncountable because it allowed for calculations and analysis of the data. The colony abundance from the 10x dilution of samples were compared to one another. (Figure 1) Wilcox test showed a p-value of 0.35. 

The number of morphotypes that I counted from the samples range from 1-6. The morophotypes of the 100x dilution samples from personal bathrooms and public restrooms were compared to one another. (Figure 2) Wilcox test showed a p-value of 0.1

The gel electrophoresis worked on all samples. Four of the six samples showed bright bands whereas the other two showed faint bands.The gel image showed no bands under the negative control. (Figure 3) The DNA used in gel electrophoresis was quanitified via Qubit analysis. (Figure 4) 

After recieving the results from Sanger Sequencing, I cleaned the sequences using Geneious. The negative control and 3H samples turned failed the sequencing. Samples 3H 1:10 and 2P 1:10 failed the sequencing, however, I found the sequences to still be usable for BLAST. Samples 2P, 1P 1:100 and 1P 1:10 were cleaned, trimmed, usable for BLAST.(Figure 5) 

The BLAST results indicated which organisms were found in my samples. (Figure 6) I found a match for three of my samples, 2P, 1P 1:100, and 1P 1:10. Two of my samples, 3H 1:10 and 2P 1:10 resulted with no match. From the blast results I found that my 2P sample matched with Bacillus lichenformis, 1P 1:10 matched with Acideovorax temperans strain R-C-TGW, and 1P 1:100 matched with Citrobacter freundii strain. 


I chose three samples 2P, 1P 1:100, and 1P 1:10 to be incorporated into phylogenies; a Bayesian phylogeny and a Maximum Likelihood phylogey. Thermus aquaticus was used as the outgroup for both phylogenies.


Bayesian phylogeny (Figure 7) 

Maximum Likelihood phylogeny (Figure 8)

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data.csv")
```

```{r filter-and-plot-abundances}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of colony abundances between Personal Bathrooms",
                "and Public Restrooms, 10x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances at different sites, 10x dilution. Despite a higher median number of colonies from Personal Bathroom samples, the mean numbers of colonies were not significantly different between the two sites. (Wilcox p = 0.35).

```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "100") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle("Comparison of morphotypes between Personal Bathrooms",
                "and Public Restrooms, 100x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 2:** Boxplot showing the number of morphotypes from the two different sites, 100x dilution. There was no difference in the mean number of morphotypes (Wilcox p=0.1).

```{r morphotypes-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "100") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

![](data/images/gel_images/2019-09-18_BIOL422_Zimmerman_culture_PCRs.jpg)

**Figure 3:** Gel image from gel electrophoresis. Image was cropped and annotated to show only my data.

| Sample   | DNA concentration (ng/uL) |   |   |   |
|----------|---------------------------|---|---|---|
| 1P 1:10  | 8.51                      |   |   |   |
| 1P 1:100 | 7.80                      |   |   |   |
| 2P       | 16.00                     |   |   |   |
| 2P 1:10  | 49.70                     |   |   |   |
| 3H 1:10  | 16.40                     |   |   |   |
| 3H 1:100 | 6.33                      |   |   |   |

**Figure 4:** DNA concentration results obtained from Qubit analysis.

| Orignial File Name      | File Name After Cleaning               | Usability | Length After Trimming | Number of Corrections |
|-------------------------|----------------------------------------|-----------|-----------------------|-----------------------|
| AS_neg_27f_G05.ab1      | AS_neg_27f_G05_AS_failed.ab1           | unusable  | 5                     | 0                     |
| AS_3H_27f_F05.ab1       | AS_3H_27f_F05_AS_failed.ab1            | unusable  | 5                     | 0                     |
| AS_3H_1_10_27f_E05.ab1  | AS_3H_1_10_27f_E05_AS_failed_blast.ab1 | usable    | 706                   | 0                     |
| AS_2P_27f_D05.ab1       | AS_2P_27f_D05_AS_cleaned.ab1           | usable    | 247                   | 3                     |
| AS_2P_1_10_27f_C05.ab1  | AS_2P_1_10_27f_C05_AS_failed_blast.ab1 | usable    | 299                   | 0                     |
| AS_1P_1_100_27f_B05.ab1 | AS_1P_1_100_27f_B05_AS_cleaned.ab1     | usable    | 574                   | 2                     |
| AS_1P_1_10_27f_A05.ab1  | AS_1P_1_10_27f_A05_AS_cleaned.ab1      | usable    | 507                   | 0                     |

**Figure 5:** Locus of each 16S sequence and its usability.

| Sample ID   | Organism                            | Query Cover | E Value   | Percent Identity | Accession  |
|-------------|-------------------------------------|-------------|-----------|------------------|------------|
| AS 1P 1:10  | Acidovorax temperans strain R-C-TGW | 99%         | 0.0       | 99.60%           | MG982481.1 |
| AS 1P 1:100 | Citrobacter freundii strain         | 100%        | 0.0       | 99.48            | MN416243.1 |
| AS 2P       | Bacillus licheniformis              | 100%        | 6*10^-118 | 98.00%           | MK737917.1 |
| AS 2P 1:10  | No good match                       | N/A         | N/A       | N/A              | N/A        |
| AS 3H 1:10  | No good match                       | N/A         | N/A       | N/A              | N/A        |
| AS 3H       | Sequence failed                     | N/A         | N/A       | N/A              | N/A        |

**Figure 6:** BLAST result metrics which indicate the microorganisms found from my samples.


![](output/phylogenies/sanger_seqs_MrBayes_phylogeny.png)

**Figure 7:** Bayesian phylogeny with thermus aquaticus as an outgroup. 

![](output/phylogenies/sanger_seqs_phyml_phylogeny.png)

**Figure 8:** Maximum likelihood phylogeny with thermus aquaticus as an outgroup.


# Discussion

My results seemed to have showed that bathroom sinks used in personal homes have a much more diverse microbiome than sinks used in public restrooms. After a day of incubation, the colony abundance from seven out of the nine culture and dilution plates from personal bathroom sinks were so high they were uncountable. I did, however, replaced the data to show that there were 1000 colonies to allow for computation and analysis. The statistic results exhibit a high p-value thus indicating that I failed to reject the null hypothesis. Therefore, my results show that they are not significant enough to prove that bathroom sinks in public restrooms will have a more diverse microbiome than the bathroom sinks in a residential home. Failure to reject the null could be due to many reasons.

One reason that may explain my results could be the times at which I obtained my data from the six locations. I collected samples from bathroom sinks from three different home as well as bathroom sinks from three different restrooms. When I obtained the samples from the bathroom sinks, I had done so in the morning after people living in the household have already finished using the bathroom to wash their hands, wash their faces, brush their teeth, etc. I obtained all my samples during the morning and went to the public restrooms right when the locations opened and therefore, it is likely that no one was able to use the restroom and sink yet that day.  

Another reason could be that the bathroom and restroom sinks that I sampled were cleaned at different times. For example, I work at Apple Stonestown and cleaners come in every few hours to deep clean the restrooms and breakroom in the store. In my house the bathroom does not get cleaned until late afternoon or at the end of the night every other day. The differences in cleaning patterns could have also affected the results of my experiment.

Three of my samples matched with an organism via BLAST. 

For this experiment I assumed that a higher number of people come into contact with sinks located in public restrooms, however, I obtained samples at a time when little to no one has yet even entered the public restroom. The bathroom sinks I sampled may also have different cleaning patterns.


# Sources Cited
