---
title: "Your Title Here"
author: Adelle Marie Sumabat
date: October 16, 2019
output:
  word_document: default
  pdf_document: default
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

Bacteria is found in high concentrations in built environments. Out of all rooms in a built environment, it has been found that bathrooms contain a higher concentration of bacteria than any other room due to bathrooms being a less ventilated room and a much more moist environment. [@ojima2002hygiene] Research has shown that the highest concentration of bacteria in residential homes are found in moist areas such as bathroom sinks. [@rusin1998reduction] Human occupancy and human contact can have an affect on the community of the microbiome of a surface area. [@ramos2014tools] 

This research is important because we spend most of our time indoors and especially in our homes. Being constantly exposed to the bacteria in our homes or public places can greatly influence our health. By knowing how much bacteria is in a certain place we come into close contact with, such as bathroom or restroom sinks, we can then find ways of reducing any contamination that could lead to possible illness. 

# Methods

## Field Sampling

To obtain the samples I used for this experiment, I swabbed the drains of bathroom sinks from three different homes as well as restroom sinks from three different public places. This totaled to six distinct sites. I used a sterile swab and inserted it into the drain of a sink. While the swab was in the drain, I moved it in a circular motion for approximately 10 seconds. I removed the swab from the sink and broke off the tip. I inserted the tip of the cotton swab into a labeled tube and set it aside for later use. I swabbed each sink twice and repeated the swabbing procedure for the sinks at each of the six sites. This allowed me to obtain a total of 12 samples. 

## Culture and Dilution Plates

Six of the twelve samples were cultured. Each sample represented a sink from each of the six sites. 

## DNA Extraction 

## Qubit Analysis

## PCR

## Gel Electrophoresis 

## Sanger Sequencing

## Data Analysis 

# Results

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data.csv")
```

```{r filter-and-plot-abundances}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of colony abundances between Personal Bathrooms",
                "and Public Restrooms, 10x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances at different sites, 10x dilution. Despite a higher median number of colonies from Personal Bathroom samples, the mean numbers of colonies were not significantly different between the two sites. (Wilcox p = 0.35).

```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "100") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle("Comparison of morphotypes between Personal Bathrooms",
                "and Public Restrooms, 100x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 2:** Boxplot showing the number of morphotypes from the two different sites, 100x dilution. There was no difference in the mean number of morphotypes (Wilcox p=0.1).

```{r morphotypes-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "100") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

![](data/images/gel_images/2019-09-18_BIOL422_Zimmerman_culture_PCRs.jpg)

**Figure 3:** 

| Sample   | DNA concentration (ng/uL) |   |   |   |
|----------|---------------------------|---|---|---|
| 1P 1:10  | 8.51                      |   |   |   |
| 1P 1:100 | 7.80                      |   |   |   |
| 2P       | 16.00                     |   |   |   |
| 2P 1:10  | 49.70                     |   |   |   |
| 3H 1:10  | 16.40                     |   |   |   |
| 3H 1:100 | 6.33                      |   |   |   |

**Figure 4:**

| Orignial File Name      | File Name After Cleaning               | Usability | Length After Trimming | Number of Corrections |
|-------------------------|----------------------------------------|-----------|-----------------------|-----------------------|
| AS_neg_27f_G05.ab1      | AS_neg_27f_G05_AS_failed.ab1           | unusable  | 5                     | 0                     |
| AS_3H_27f_F05.ab1       | AS_3H_27f_F05_AS_failed.ab1            | unusable  | 5                     | 0                     |
| AS_3H_1_10_27f_E05.ab1  | AS_3H_1_10_27f_E05_AS_failed_blast.ab1 | usable    | 706                   | 0                     |
| AS_2P_27f_D05.ab1       | AS_2P_27f_D05_AS_cleaned.ab1           | usable    | 247                   | 3                     |
| AS_2P_1_10_27f_C05.ab1  | AS_2P_1_10_27f_C05_AS_failed_blast.ab1 | usable    | 299                   | 0                     |
| AS_1P_1_100_27f_B05.ab1 | AS_1P_1_100_27f_B05_AS_cleaned.ab1     | usable    | 574                   | 2                     |
| AS_1P_1_10_27f_A05.ab1  | AS_1P_1_10_27f_A05_AS_cleaned.ab1      | usable    | 507                   | 0                     |

**Figure 5:**

| Sample ID   | Organism                            | Query Cover | E Value   | Percent Identity | Accession  |
|-------------|-------------------------------------|-------------|-----------|------------------|------------|
| AS 1P 1:10  | Acidovorax temperans strain R-C-TGW | 99%         | 0.0       | 99.60%           | MG982481.1 |
| AS 1P 1:100 | Citrobacter freundii strain         | 100%        | 0.0       | 99.48            | MN416243.1 |
| AS 2P       | Bacillus licheniformis              | 100%        | 6*10^-118 | 98.00%           | MK737917.1 |
| AS 2P 1:10  | No good match                       | N/A         | N/A       | N/A              | N/A        |
| AS 3H 1:10  | No good match                       | N/A         | N/A       | N/A              | N/A        |
| AS 3H       | Sequence failed                     | N/A         | N/A       | N/A              | N/A        |

**Figure 6:**


![](output/phylogenies/sanger_seqs_MrBayes_phylogeny.png)

**Figure 7:**

![](output/phylogenies/sanger_seqs_phyml_phylogeny.png)

**Figure 8:**


# Discussion

# Sources Cited
