---
title: No Significant Differences Found Between The Microbiome of Personal Bathroom Sinks and Public Restroom Sinks
author: Adelle Marie Sumabat
date: October 16, 2019
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

Bacteria is found in high concentrations in built environments. Building design and its use has an affect on the type of microbial communities found in residential surfaces [@peccia2016buildings]. For example, a building that is used as a factory with a high number of occupants, animals, or crops found within it will have a different microbial community than a building that is kept sanitized such as a hospital. Some key factors that have been found to greatly affect the microbial community in residential buildings include how moist the environment is, the age of the building, and whether or not animals are found within the building [@kettleson2015key]. Out of all rooms in a built environment, it has been found that bathrooms contain a higher concentration of bacteria than any other room due to bathrooms being a less ventilated room and a much more moist environment [@ojima2002hygiene]. In addition to the bacteria on surfaces in bathrooms, there is also a higher risk of bacteria exposed to humans via inhalation due to its moist environment [@azuma2013assessing]. In fact, research has shown that antibiotic-resistant _Staphylococcus_ bacteria are commonly found in indoor air of residential buildings and thrive in the moist environments of bathrooms [@rintala2008diversity]. It has also been found that skin associated bacteria are typically found on surfaces that have been touched by humans. Gut associated bacteria as well as fecal contaminants are typically found on the surface of toilets. This shows that human occupancy and human contact can have an affect on the microbial community within a surface area [@ramos2014tools]. Microorganisms are found all over our skin, gut, mucus, and covers all the rest of our body's surface. It was been found that our bodies contain at least 10 times more bacterial cells than human ones. This information also shows that bacteria doesn’t necessarily grow on surfaces found in built environments, but instead gets transferred onto it. Therefore, bacteria could be easily transmitted by means of simply touching a surface in a bathroom or restroom.

Bacteria play many roles that are beneficial to humans. They can synthesize vitamins, break down food into absorbable nutrients, and stimulate our immune systems. However, there are still many types of bacteria that have the ability to cause infections that may result in deadly disease or death itself. Bacteria have also been successful organisms because they have great adaptive capabilities. With that being said, many pathogenic bacteria have evolved to become resistant to drugs and antibiotics. Some bacteria are naturally resistant to antibiotics and other bacteria can achieve resistance by having the resistance gene trasnferred to their plasmid from the plasmid of an antibiotic resistant drug. In addition, most bacteria divde every 20 minutes and thus are able to transfer their antibiotic resistance gene to their offspring. Because of the increasing rate of antibiotic resistance, many infections such as pneumonia, tuberculosis, gonorrhea, and many more are becoming harder to treat [@alanis2005resistance]. It is important to be aware of when we are exposed to pathogenic bacteria because we never know if they carry the anitbiotic resistant gene.


Research has shown that the overall highest concentration of bacteria in residential homes are found in moist areas such as bathroom sinks [@rusin1998reduction]. For this project I attempted to answer the following question, “Does the number of people getting in contact with a bathroom sink affect the diversity of the microbiome found within it?” I hypothesized that bathroom sinks in public restrooms will have a more diverse microbiome than the bathroom sinks in a residential home because of the higher number of people who have access to the public restrooms.

I have analyzed the microbiome of bathroom and restroom sinks. I compared the microbiome of bathroom sinks from residential houses to the microbiome of sinks in public restrooms. I used culture and dilution plates to obtain colony and morphotype abundance from each sink. Sequencing technology has been used in previous research to investigate indoor microbial communities [@flores2011microbial] and thus, I used Sanger sequencing and BLAST analysis to identify the organisms found from my samples. I further analyzed my results through the use of a software called Geneious Prime. Although culture and dilution plates showed that personal bathrooms have a higher abundance and diversity in bacteria, statistical results showed that my results were not significant and therefore were not enough to prove my hypothesis to be true.

Research has shown that next generation sequencing can provide insight into microbial genomes, the structure of the communities they inhabit, and their impact on human health and diseases [@10.1093/bfgp/els062]. I used Illumina next generation sequencing to provide an even broader range of information on the microbial communities and organisms from my environmental samples. Using an operating system called Bash command line, I was able to analyze the results from the Illumina sequence to gather more information on the organisms found within my environmental samples as well as culture samples. I created scripts in Bash to sort and analyze through the data that resulted from the Illumina sequencer. I found thousands of sequences in all but one environmental sample. The negative and positive controls also had thousands of sequences found within in. Most of the organisms found in my culture samples were, unexpectedly, not found in my environmental sample. This could mostly be due to contamination that may have taken place during the preparation for Illumina sequencing. I also used DADA2 and phyloseq for further analysis of the data I was able to obtain from the Illumina sequencing. This allowed for me to visualize my results for better analysis.

This research is important because we spend most of our time indoors and especially in our homes. Being constantly exposed to the bacteria in our homes or public places can greatly influence our health. In fact, it has been found that roughly one in five human deaths are caused by pathogenic disease [@dunn2010global]. Previous literature has also found that there are roughly around 1415 species of infectious organisms that are pathogenic to humans [@taylor2001risk]. In order to prevent infection, we must first know the source of the bacteria and how humans come into contact with them. By knowing the structures of the microbial communities in everyday environments we can find ways of eliminating the bacteria that cause disease. However, there has been research conducted that shows a relationship between antibiotic-resitant microbial communities on residential surfaces and methods of keeping the surfaces clean such as disinfectant sprays, natural products, etc [@rutala2000antimicrobial]. With that being said, there are emerging questions to follow up with such as whether or not antibacterial cleaning products can be correlated to the emerging number of drug-resistant bacteria [@aiello2005antibacterial]. By knowing how much bacteria is in a certain place we come into close contact with, such as bathroom or restroom sinks, we can then find ways of reducing any contamination that could lead to possible illness as well us come up with better ways to maintain proper hygiene within a restroom.

# Culture Methods

## Field Sampling

To obtain the samples I used for this experiment, I used sterile cotton swabs, sterile buffer, sterile gloves, and tubes. I swabbed the drains of bathroom sinks from three different residential homes as well as restroom sinks from three different public places. This totaled to six distinct sites. I used a sterile swab and inserted it into the drain of a sink and moved it in a circular motion for approximately 10 seconds. I repeated the swabbing procedure for all the sinks at each of the six sites. This allowed me to obtain a total of 12 samples. I labeled the tubes from personal bathrooms as 1H, 2H, and, 3H. The tubes from public restrooms were labeled as 1P, 2P, and 3P. I used six samples, one sample from each location, for culturing. The other six environmental samples were set aside and frozen at -20 degrees Celsius and were later used for Illumina sequencing.

## Culture and Dilution Plates

I cultured and diluted the samples I obtained from the six different sites. The plates I used for culturing were made up of tryptic soy agar (TSA) medium on 100mm petri dishes. I added a sterile buffer, PBS, to one of the tubes containing a swab and vortexed the tube. I created 1:1, 1:10, and 1:100 diluted solutions and transferred them to culture plates and spread with beads. I repeated this process with each of the six samples and obtained a total of 18 culture and dilution plates. The plates were incubated overnight at 37 degrees Celsius. The following day, the number of colonies and morphotypes were counted and documented.

## DNA Extraction 

After the colonies and morphotypes were counted and documented, I chose six different colonies to extract DNA from. I labeled tubes for each sample. I added cells from one colony to a tube using a sterile pipette tip. I then followed the manufacturer’s protocol from the Sigma REDExtract-N-Amp kit. I then analyzed the DNA via Qubit analysis to quantify the DNA concentrations.

## PCR

For PCR, I had to first make the master mix. The volumes of reagents required for the mix for one reaction were 10 uL AMP, 0.8 uL 27f primer, 0.8 uL 1492r primer, 1 uL BSA, 1 uL template DNA and 6.4 uL H20. After I created the master mix, I divided the master mix into 7 tubes. 1 uL of DNA from each sample was added to a tube. I added H20 to the seventh tube to serve as the negative control.

The PCR cycle conditions began with five minutes of being in 95 degrees Celsius. I then set the thermocycler to undergo a cycle that started off at 94 degrees Celsius for 30 seconds, 65 degrees Celsius for 30 seconds, and 72 degrees Celsius for one minute. This was repeated for ten cycles and the temperature was stepped down by 1 degree per cycle. I changed the thermocycler again and set the temperature to 94 degrees Celsius for 30 seconds, 55 degrees Celsius for 30 seconds, and 72 degrees Celsius for 1 minute. This was repeated for 25 cycles. I then changed the thermocycler temperature to 72 degrees Celsius for 10 minutes and the thermocycler was finally held at 4 degrees Celsius.


## Gel Electrophoresis 

The gel I used for my experiment contained 2% agarose gel and SYBR safe dye. 4 uL of our DNA samples were loaded onto a 50 well gel. I used ladders to serve as a positive control at each end of the wells. I ran the gel under 140V and 1X TAE buffer conducted current through the gel.

## Sanger Sequencing

After checking for successful amplification using gel electrophoresis, PCR products were cleaned with ExoSAP (Invitrogen) and sent for unidirectional Sanger sequencing at MCLAB (South San Francisco, CA) using the 27f primer.


## Data Analysis 

The alignment program I used to analyze the Sanger sequences is a software called Geneious Prime version 2019.2.3. I received the results from Sanger sequencing as a zip file. I imported my files to Geneious and was able to analyze my sequences. I cleaned up and trimmed the sequences that I confirmed to be usable for BLAST and used IUPAC ambiguity codes for bases that were hard to identify. I confirmed that any sequences that were similar to the negative control were unusable for BLAST. For the rest of the sequences that were usable for BLAST, I cut off the beginning and ends of the sequences due to the noise in those sections of the reads. I used IUPAC ambiguity codes to replace the low-quality nucleotides. 

I used a software called BLAST to match the sequences from my samples to organisms that were in NCBI's nucleotide database. Multiple organisms were shown from the BLAST results and I chose the top result for each sample. I then used Geneious to create a Bayesian phylogeny and a Maximum Likelihood phylogeny. I created the phylogenies using multiple sequence alignments generated using MAFFT [@katoh2009multiple]. I used the default parameters for MAFFT. I used MrBayes software in Geneious Prime to create a Bayesian phylogeny [@huelsenbeck2001mrbayes]. For the parameters, I used GTR for the substitution model, _Thermus aquaticus_ was used as an outgroup, and I used the default for the rest of the parameters. I used PhyML software to create a Maximum Likelihood phylogeny [@guindon2005phyml]. For the parameters, I used GTR for the substitution model, Bootstrap as branch support, _Thermus aquaticus_ was used as an outgroup, and then I used the default for the rest of the parameters. My data reached the minimum of three sequences that was required in order to successfully create both phylogeny trees.

# Culture Free Methods For Environmental Samples

## DNA Extraction

After the initial six samples were sequenced via Sanger sequencing, the other six were taken out of the freezer to be prepared for Illumina sequencing. I followed the manufacturer’s protocol from the Sigma REDExtract-N-Amp kit. I then analyzed the DNA via Qubit analysis to quantify the DNA concentrations.

## PCR

To prepare the samples for PCR, I calculated the reagents required for the master mix. The reagents required for the master mix for one reaction was 10 uL AMP, 0.8 uL 16SF and 0.8 uL 16SR primers, 1 uL BSA, 1 uL DNA template, and 6.4 uL H2O. After I created the master mix, I divided the master mix into 7 tubes. 1 uL of DNA from each sample was added to a tube. I added H20 to the seventh tube to serve as the negative control.

The cycle began with three minutes of being in 96 degrees Celsius. The thermocycler was changed to undergo 95 degrees Celsius for 30 seconds, 55 degrees Celsius for 30 seconds, and 72 degrees Celsius for 30 seconds. This was repeated for 25 cycles. The thermocycler was then set to 72 degrees Celsius for 5 minutes before being held at 4 degrees Celsius.

## Gel Electrophoresis

The gel I used for my experiment contained 2% agarose gel and SYBR safe dye. 4 uL of our DNA samples were loaded onto a 50 well gel. I used ladders to serve as a positive control at each end of the wells. I ran the gel under 140V and X TAE buffer conducted current through the gel.

## Illumina Sequencing

Following electrophoresis, PCR products were purified using AmpureXP magnetic beads (Beckman-Coulter) and quantitated using a PicoGreen fluorescent assay (Invitrogen) on a Tecan Infinite M Plex plate reader. Purified PCR products were used as the template for a second round of PCR, which served to attach unique pairs of forward and reverse Illumina barcodes (Nextera XT Index 2 kit). All other components of the PCR mixtures were as for the first-round PCR. These reactions were cycled at 95 degrees C for 3 minutes, then 8 cycles of: 95 degrees C for 30 s, 55 degrees C for 30s, and 72 degrees C for 30s, followed by a 5 minute elongation cycle at 72 degrees Following this amplification, PCR products were purified and normalized with a SequelPrep normalization plate (Invitrogen), pooled, and then quantified once again with a Qubit 4 fluorometer (Invitrogen) and a TapeStation 4200 (Agilent) to verify library size and concentration. This library was then diluted to the loading concentration (50 pM) and combined with an Illumina PhiX spike-in library (5% spike in) and sequenced on an Illumina iSeq using a 2 x 150 bp consumable cartridge.

## Data Analysis

I created bash scripts to assess the quality of the data, toss out bad sequences, clean other sequences, convert files to the correct format, and finally BLAST the data in order to get information on the organisms present in my environmental samples. The sequences were trimmed and cleaned to provide more accurate DNA sequences from my environmental samples. Trimmomatic was used to cut the first five and last five base pairs, cut any base pair with a quality score below 25, as well as cut any sequences that contained less than 140 base pairs. The files needed to be converted from fastq to fasta format because BLAST analysis only works with fasta formatted files. The organism match was based off NCBI’s nucleotide database and I used the single top match.

DADA2 [@callahan2016dada2] and phyloseq [@mcmurdie2012phyloseq] were also used to analyze the environmental samples. DADA2 pipeline was used to process my subsampled 16S fastq files. It was used to filter, trim, generate error models, get rid of duplicated samples, denoise sequences, check for and remove chimeras, and assign taxonomy to the sequences. DADA2 was also used to create a phyloseq object that was used for analysis and visualization through packages in R markdown called pyhloseq and dplyr.

# Results

## Culture Samples

Results from the bacteria cultures showed that personal bathrooms contained too many colonies to count. For the purpose of this experiment, I documented that I counted 1000 colonies from the plates that were uncountable. I used  a high value to represent the high abundance of colonies because it allowed for calculations and analysis of the data. The colony abundance from the 10x dilution of samples from each of the sites were compared to one another (Figure 1). Wilcox test showed a p-value of 0.35. The number of morphotypes that I counted from the samples ranged from 1-6. The morophotypes of the 100x dilution samples from personal bathrooms and public restrooms were compared to one another (Figure 2). Wilcox test showed a p-value of 0.1

The gel electrophoresis worked on all samples. Four of the six samples showed bright bands whereas the other two showed faint bands.The gel image showed no bands under the negative control (Figure 3). The DNA used in gel electrophoresis was quantified via Qubit analysis (Figure 4).

After receiving the results from Sanger sequencing, I cleaned the sequences using Geneious. The negative control and 3H samples failed and were unusable for BLAST. Samples 3H 1:10 and 2P 1:10 failed the sequencing, however, I found the sequences to still be usable for BLAST. Samples 2P, 1P 1:100 and 1P 1:10 were cleaned, trimmed, and usable for BLAST (Figure 5).

The BLAST results indicated which organisms were found in my samples (Figure 6). I found a match for three of my samples, 2P, 1P 1:100, and 1P 1:10. Two of my samples, 3H 1:10 and 2P 1:10 resulted with no match. From the blast results I found that my 2P sample matched with _Bacillus licheniformis_, 1P 1:10 matched with _Acidovorax temperans_ strain R-C-TGW, and 1P 1:100 matched with _Citrobacter freundii_ strain.


I chose three samples 2P, 1P 1:100, and 1P 1:10 to be incorporated into phylogenies; a Bayesian phylogeny and a Maximum Likelihood phylogeny. _Thermus aquaticus_ was used as the outgroup for both phylogenies. Bayesian phylogeny (Figure 7) was created to show the relationship between organisms. The Bayesian posterior probability was 0.9912. AS 1P 1:10 and AS 1P 1:100 were grouped together in a monophyletic clade. Maximum Likelihood phylogeny (Figure 8) was created to show the relationship between organisms. The Maximum Likelihood bootstrap value was 99. AS 1P 1:10 and AS 1P 1:100 were grouped together in a monophyletic clade in this phylogeny tree as well.

## Environmental Samples

Qubit analysis confirmed that there was enough DNA concentrations to sequence in all six of my environmental samples. Results from the Illumina sequencer showed many sequences found within each environmental sample. Prior to trimming, the sample labeled 1H was found to have a total of 6389 sequences, 1P had 8816 sequences, 2H had 4581 sequences, 2P had 14 sequences, 3H had 10579 sequences, and 3P had 5808 sequences. The negative control sample labeled neg1 had 10990 sequences, neg2 has 3290 sequences, neg3 had 13837 sequences, neg4 had 7825 sequences. The positive control labeled zymo1 had 5236 sequences prior to trimming, zymo2 had 4139 sequences, zymo3 had 8448 sequences, and zymo4 had 9287 sequences. After trimming the sequences from the samples 1H ended up having 6137 sequences, 1P had 8463 sequences, 2H had 4404 sequences, 2P had 4 sequences, 3H had 10148 sequences, and 3P had 5552 sequences. 

Based on the fastqc reports, the quality of my environmental sample sequences were high. For all six environmental samples, the quality was shown to be at its lowest at the 150-151 position or the end of the sequences. However, it still portrayed high quality based on the "Per base sequence quality" chart. The quality of the sequences were confirmed after trimming the sequences via Bash command line. Out of the thousands of sequences found in each sample, only a few hundred sequences from each sample were "thrown out" after running the code to trim and clean the sequences. The environmental sample from site 2P had the lowest count of sequences with a count of 14.

_Acidovorax temperans_ was found in the 1P environmental sample along with 71 other strains of _Acidovorax_. _Citrobacter freundii_ was not found in the 1P environmental sample.  _Bacillus licheniformis_ was not found in the 2P environmental SAMPLE.

According to my BLAST results that matched with organisms from the NCBI's nucleotide database, _Acinetobacter parvus_ was the most abundant species in the 1P environmental sample with 1573 sequences. _Enterococcus faecium_ was most abundant in the 2P environmental sample with 2 sequences. _Sphingomonas yabuuchiae_ was most abundant in the 3P environmental sample with 644 sequences. _Acinetobacter parvus_ was also most abundant in the 1H environmental sample with 534 sequences. _Mycobacteroides franklinii_ was most abundant in the 2H environmental sample with 567 sequences. _Serratia liquefaciens_ was most abundant in the 3H environmental sample with 2075 sequences.

Overall, the BLAST results show that _Acinetobacter tandoii_, _Salmonella enterica_, and _Listeria monocytogenes_ were found to be the most abundant organisms within the environmental samples.

DADA2 results show that _Proteobacteria_, _Actinobacteria_, and _Firmicutes_ were the top three phyla that dominated in abundance across my samples (Figure 11). _Acinetobacter_, _Acidovorax_, and _Pseudomonas_ were the top three most abundant genus found within the environmental samples (Figure 14).

```{r load-libraries}
# Load packages for analysis
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
library("lubridate")
library("forcats")
library("dada2")
library("phyloseq")
```

```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv(
  "data/raw_data/raw_culture_count_data/BIOL422_culturing_data.csv")
```

```{r filter-and-plot-abundances}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of colony abundances between Personal Bathrooms",
                "and Public Restrooms, 10x dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 1:** Boxplot of colony abundances at different sites, 10x dilution. Despite a higher median number of colonies from Personal Bathroom samples, the mean numbers of colonies were not significantly different between the two sites. (Wilcox p = 0.35).

```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "100") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle("Comparison of morphotypes between Personal Bathrooms",
                "and Public Restrooms, 100x dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 2:** Boxplot showing the number of morphotypes from the two different sites, 100x dilution. There was no difference in the mean number of morphotypes (Wilcox p=0.1).

```{r morphotypes-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Adelle Sumabat") %>%
  filter(dilution_1_10_or_100 == "100") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

![](data/images/gel_images/2019-09-18_BIOL422_Zimmerman_culture_PCRs.jpg)

**Figure 3:** Gel image from gel electrophoresis. Image was cropped and annotated to show only my data.

| Sample   | DNA concentration (ng/uL) |   |   |   |
|----------|---------------------------|---|---|---|
| 1P 1:10  | 8.51                      |   |   |   |
| 1P 1:100 | 7.80                      |   |   |   |
| 2P       | 16.00                     |   |   |   |
| 2P 1:10  | 49.70                     |   |   |   |
| 3H 1:10  | 16.40                     |   |   |   |
| 3H 1:100 | 6.33                      |   |   |   |

**Figure 4:** DNA concentration results obtained from Qubit analysis. 2-50 ng/uL was considered to be a good range for DNA concentration.

| Orignial File Name      | File Name After Cleaning               | Usability | Length After Trimming | Number of Corrections |
|-------------------------|----------------------------------------|-----------|-----------------------|-----------------------|
| AS_neg_27f_G05.ab1      | AS_neg_27f_G05_AS_failed.ab1           | unusable  | 5                     | 0                     |
| AS_3H_27f_F05.ab1       | AS_3H_27f_F05_AS_failed.ab1            | unusable  | 5                     | 0                     |
| AS_3H_1_10_27f_E05.ab1  | AS_3H_1_10_27f_E05_AS_failed_blast.ab1 | usable    | 706                   | 0                     |
| AS_2P_27f_D05.ab1       | AS_2P_27f_D05_AS_cleaned.ab1           | usable    | 247                   | 3                     |
| AS_2P_1_10_27f_C05.ab1  | AS_2P_1_10_27f_C05_AS_failed_blast.ab1 | usable    | 299                   | 0                     |
| AS_1P_1_100_27f_B05.ab1 | AS_1P_1_100_27f_B05_AS_cleaned.ab1     | usable    | 574                   | 2                     |
| AS_1P_1_10_27f_A05.ab1  | AS_1P_1_10_27f_A05_AS_cleaned.ab1      | usable    | 507                   | 0                     |

**Figure 5:** Locus of each 16S sequence, its usability, and sequence length after trimming.

| Sample ID   | Organism                            | Query Cover | E Value   | Percent Identity | Accession  |
|-------------|-------------------------------------|-------------|-----------|------------------|------------|
| AS 1P 1:10  | _Acidovorax temperans_ strain R-C-TGW | 99%         | 0.0       | 99.60%           | MG982481.1 |
| AS 1P 1:100 | _Citrobacter freundii_ strain         | 100%        | 0.0       | 99.48            | MN416243.1 |
| AS 2P       | _Bacillus licheniformis_              | 100%        | 6*10^-118 | 98.00%           | MK737917.1 |
| AS 2P 1:10  | No good match                       | N/A         | N/A       | N/A              | N/A        |
| AS 3H 1:10  | No good match                       | N/A         | N/A       | N/A              | N/A        |
| AS 3H       | Sequence failed                     | N/A         | N/A       | N/A              | N/A        |

**Figure 6:** BLAST result metrics which indicate the microorganisms found from my samples. AS 2P 1:10 and 3H 1:10 were failed sequences, however, I still considered them to be usable for BLAST. BLAST showed no match for the two samples. AS 3H was a failed sequence and did not undergo BLAST.


![](output/phylogenies/sanger_seqs_MrBayes_phylogeny.png)

**Figure 7:** Bayesian phylogeny with _Thermus aquaticus_ as an outgroup. Bayesian posterior probability of 0.9912

![](output/phylogenies/sanger_seqs_phyml_phylogeny.png)

**Figure 8:** Maximum likelihood phylogeny with _Thermus aquaticus_ as an outgroup. Maximum Likelihood bootstrap value of 99.

```{r load-data}
# load in the saved phyloseq object to work with
load("output/phyloseq_obj.Rda")
```

```{r alpha-diversity-metrics}
# alpha diversity metrics
plot_richness(phyloseq_obj,
              x = "type",
              measures = c("Shannon")) +
  xlab("Type of sample") +
  geom_boxplot() +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45,
                                   hjust = 1))
```

**Figure 9.** According to Shannon Diversity Index, personal bathrooms and public restrooms have a similar species diversity and evenness mean. However, personal bathrooms have an overall higher species diversity and evenness compared to public restrooms. 

```{r plot-of-taxa}
##########################################
# dplyr and ggplot analyses
##########################################

# melt phyloseq obj into a data frame for dplyr/ggplot
# analysis and visualization
melted_phyloseq <- psmelt(phyloseq_obj)

# turn all factor columns into character columns for dplyr
melted_phyloseq <- melted_phyloseq %>%
  mutate_if(is.factor, as.character)

# create a summary table of sequence counts for each Phylum
melted_phyloseq %>%
  group_by(Phylum) %>%
  summarize(sum_abundance = sum(Abundance,
                                  na.rm = TRUE))
# bar plot of taxa
ggplot(melted_phyloseq,
       aes(x = type,
           y = Abundance,
           color = Phylum,
           fill = Phylum)) +
  geom_col(position = position_fill()) +
  theme(axis.text.x = element_text(angle = 45,
                                   hjust = 1)) +
  xlab("Type of Sample") +
  ylab("Number of Sequences") +
  ggtitle("Number of Sequences")
```

**Figure 10:** The number of sequences found present within the samples organized by the phylum to which the organism belongs to. 

```{r summary-table}
# Create a summary table of sequence results for each Phylum and its abundance
melted_phyloseq %>%
  group_by(Phylum) %>%
  summarize(sum_abundance = sum(Abundance,
                                na.rm = TRUE)) %>%
  ggplot(aes(x = Phylum,
             y = sum_abundance)) +
  ggtitle("Phylum Abundance") +
  labs(y = "Sum Abundance") +
  geom_col() +
  theme(axis.text.x = element_text(angle = 45,
                                   hjust = 1,
                                   vjust = 1))
```

**Figure 11:** The overall abundance of organisms by phyla.

```{r asv-richness-across-samples}
melted_phyloseq %>%
  filter(grepl("AS", Sample)) %>%
  filter(Abundance > 0) %>%
  group_by(type, Sample, Phylum) %>%
  tally() %>%
  ggplot(aes(x = Sample,
             y = n,
             fill = Phylum)) +
  geom_col() +
  facet_wrap(~type,
             scales = "free_x")
```

**Figure 12:** Comparison of phylum abundance between personal bathrooms and public restrooms. 2P environmental sample site was not taken into consideration for this figure.

```{r abundance-of-Acinetobacter} 
melted_phyloseq %>%
  filter(grepl("AS", Sample)) %>%
  filter(Genus == "Acinetobacter") %>%
  filter(Abundance > 0) %>%
  group_by(Sample) %>%
  summarize(sum_abund = sum(Abundance,
                            na.rm = TRUE)) %>%
  kable()
```

**Figure 13:** Abundance of _Acinetobacter_ across samples.

```{r top-genera}
melted_phyloseq %>%
  filter(grepl("AS", Sample)) %>%
  filter(Abundance > 0) %>%
  group_by(Genus) %>%
  summarize(sum_abund = sum(Abundance,
                            na.rm = TRUE)) %>%
  arrange(desc(sum_abund)) %>%
  kable()
```

**Figure 14:** Abundance of organisms present in samples by genus.

# Discussion

For this experiment I intended to determine if the number of people getting in contact with a bathroom sink affects the diversity of the microbiome found within it. After some research, I hypothesized that bathroom sinks in public restrooms will have a more diverse microbiome than the bathroom sinks in a residential home because of the higher number of people who have access to it.

The colony and morphotype abundance from my samples have showed that bathroom sinks used in personal homes have a much more diverse microbiome than sinks used in public restrooms. After a day of incubation, the colony abundance from seven out of the nine culture and dilution plates from personal bathroom sinks were so high that they were uncountable. I did, however, replace the data to show that there were 1000 colonies to allow for computation and analysis. However, statistic results exhibited a high p-value for both colony abundance and number of morphotypes thus indicating that I failed to reject the null hypothesis. Therefore, my results show that they are not significant enough to prove that bathroom sinks in public restrooms will have a more diverse microbiome than the bathroom sinks in a residential home. Failure to reject the null could be due to many reasons.

One reason that may explain my results could be the time at which I obtained my samples from the six locations. I collected samples from bathroom sinks from three different homes as well as bathroom sinks from three different restrooms. When I obtained the samples from the bathroom sinks representing sinks from residential homes, I had done so in the morning after people living in the household have already finished using the bathroom to wash their hands, wash their faces, brush their teeth, etc. I also went to the public restrooms right when the locations opened and therefore, it is likely that no one was able to use the restroom and sink yet that day.

Another reason could be that the bathroom and restroom sinks that I sampled were cleaned at different times. For example, I work at Apple Stonestown and cleaners come in every few hours to deep clean the restrooms and break room in the store. In my house, the bathroom does not get cleaned until late afternoon or at the end of the night every other day. The differences in cleaning patterns could have also affected the results of my experiment.

For this experiment I assumed that a higher number of people come into contact with sinks located in public restrooms, however, I obtained samples at a time when little to no one has yet even entered the restroom. Although more people have access to public restrooms, it does not necessarily mean they actually get in contact with it. The bathroom sinks I sampled may also have different cleaning patterns.

Three of my samples matched with an organism via BLAST. My sample labeled 2P matched with _Bacillus licheniformis_. After further research I found that _Bacillus licheniformis_ is commonly found in soil and bird feathers. It is not a human pathogen. _Bacillus lichenformis_ actually plays a role in nutrient cycling. A peer reviewed article explains that this bacteria is used in the biotechnology industry to manufacture enzymes, antibiotics, biochemicals and consumer products [@rey2004complete]. It is interesting to see that this organism matches with my sample 2P. I took this sample from a public restroom at Apple Stonestown. Although Apple is a technology company, it does not contribute to biotechnology so it did not make sense to come across this organism at that location. It is possible, however, that a person that works in biotechnology and is exposed to this organism came to the store and washed his or her hands in the restroom sink.

The sample labeled 1P 1:10 matched with _Acidovorax temperans_ strain R-C-TGW, a gram negative bacterium that is known to cause disease in plants. This bacteria is normally found in soil and water environments. It is known to cause disease in plants, mostly watermelons and honeydews. When plants become infected by this bacteria, brown blotches begin to appear on the surface of leaves and fruits.

The sample labeled 1P 1:100 matched with _Citrobacter freundii_ strain, a Gram-negative bacteria that is comonly found in a healthy human gut and other strains of this bacteria play important roles in nitrogen fixation in soil microbiomes. Most strains of _Citrobacter freundii_ are beneficial to humans, however, some strains of this bacteria are known to cause a number of “opportunistic infections.” This means that this bacteria does not necessarily infect healthy people, but it infects people with previously weakened immune systems. Those who have a weak immune system and become infected by _Citrobacter freundii_ are prone to urinary tract infections,  intestinal infection, infections in respiratory tract, blood, or meningitis [@ranjan2013citrobacter]. Research has shown that some _Citrobacter_ strains that can cause meningitis and brain abscess. It is actually quite alarming to see this bacteria present in my samples because experiments show that it is capable of crossing the blood-brain barrier and thus is able to invade and replicate within brain cells [@badger1999citrobacter]. 

1P 1:10 and 1P 1:100 are samples I obtained from swabbing a public restroom sink in Burger King. Burger King sells food, therefore, I am able to make sense as to how and why a bacteria that causes disease in crops was found in the sink. However, it surprises and also worries me to see that bacteria that can cause many kinds of infections is present in a public restroom sink. I later found that the Burger King that I took a sample from was also right next to a gas station that had no restrooms. It is possible this Burger King may have a higher number of users than normal being next to a busy gas station that provided no restrooms for its customers.

For the results I received back from Sanger sequencing, three of my samples were sequenced successfully. I was able to clean, trim, and match the samples to organisms using BLAST. Two of the samples failed meaning that the sequences I obtained were of extremely low quality but were still usable for BLAST, however, BLAST analysis showed no match for those samples. The sequences could have just been too noisy for BLAST to be able to match an organism to it. One of my samples completely failed and was unusable. The sequence looked exactly like that of the negative control. For the three sequences that failed, it could have been due to either contamination of the samples somewhere throughout the procedure. The samples could have also contained DNA from more than one individual and thus resulted in messy sequences that had no match in the NCBI nucleotide database. Throughout the experiment, I also found that some of the pipettes that I used were calibrated a bit differently from others. That caused me to redo certain steps throughout the  experiment which may also have potentially affected my data. It is also possible that I could have been using contaminated pipette tips throughout certain parts of the procedure.

According to the fastqc report, the environmental samples contained thousands of high quality sequences with the exception of the sample from site 2P which had the lowest count of 14 sequences. As mentioned earlier, the 2P sample was from a restroom at Apple Stonestown which has cleaners coming in almost every hour to clean and disinfect the restrooms, break room, etc. Since I came in the morning, the restroom was still fresh from the morning cleaners. This can explain the significantly low amount of sequences found in the sample taken from this site. 

The results from the Illumina sequences were not as I had expected. I found thousands of sequences from different organisms within each sample, including the positive and negative controls. Because Illumina sequencing is suppose to extract essentially all the organisms found in a sample, whatever I found in my culture samples should also be found in my environmental samples. Although, _Acidovorax temperans_ was indeed found in the environmental sample labeled 1P, _Citrobacter freundii_ was not. Both organisms were found in my cultured 1P samples via BLAST analysis. This tells me that _Acidovorax temperans_ is a bacteria that was found and obtained from the restroom sink from the 1P site and _Citrobacter freundii_ could be a contaminant in my culture samples. There were 71 strains of _Acidovorax_ found in my 1P environmental sample which gave me more assurance that the bacteria is truly obtained from the environmental sample and not a contaminant.  _Bacillus lichenformis_ was another organism found from my culture sample from the 2P site that was not found in the environmental sample. This again shows that _Bacillus lichenformis_ is actually an organism that may have contaminated my culture sample from the 2P site at some point during my procedure. I found thousands of sequences from different organisms within each sample, including the positive and negative controls. Even after trimming, the positive and negative controls exhibited a high number of sequences in the thousands range. The sequences found in the controls could possibly be due to contamination that must have taken place sometime during the preparation for the sequencer. If this experiment and procedures were to be repeated, I recommend taking caution to avoid any contamination among the samples.

BLAST results showed that _Acinetobacter parvus_ was the most abundant species in the 1P environmental sample with 1573 sequences and in the 1H environmental sample with 534 sequences. _Acinetobacter_ bacteria are commonly found in humans, however, can be responsible for a life-threatening disease called sepsis in patients in intensive care units. _Acinetobacter parvus_ is different from other _Acinetobacter_ strains because it produces significantly smaller colonies than the norm. _Acinetobacter_ are naturally antibiotic resistant and can transfer their resistance gene to other bacteria via horizontal gene transfer. _Enterococcus faecium_ was most abundant in the 2P environmental sample with 2 sequences. _Enterococcus faecium_ is a Gram-positive bacterium. It can be innocuous and is typically found in the gastrointestinal tract of animals and humans. _Enterococcus faecium_ also has the potential to be pathogenic and can cause diseases such as neonatal meningitis or endocarditis. It has developed multi-drug antibiotic resistance as well as resistance to alcohol-based disinfectants. This bacteria can cause infections of the bloodstream, urinary tract infections, and wound infections associated with catheters or surgery. _Enterococcus faecium_ was initially considered to be part of the genus _Streptococcus_. This bacteria is known to infect people with already weakened immune systems, typically hospitalized patients. _Enterococcus faecium_ is believed to be one of the top three leading causes of hospital-acquired infection. _Sphingomonas yabuuchiae_ was most abundant in the 3P environmental sample with 644 sequences. _Sphingomonas_ bacteria are Gram-negative rod shapex bacteria that have the ability to ability to survive in low concentrations of nutrients. Some strains of this bacteria can cuase non-life threatening infections that can easily be treated with antibiotics. It has biodegradative and biosynthetic capabilities which allows biotechnology to use it to produce polymers used in the food industry. _Mycobacteroides franklinii_ was most abundant in the 2H environmental sample with 567 sequences. _Mycobacteroides_ bacteria are pathogenic to humans. Although I was unable to find any articles or journals online about _Mycobacteroides franklinii_, I found other strains such as _Mycobacterium abscessus_ which is a rapidly growing multi-drug antibiotic resistant pathogen that causes chronic lung infection as well as skin and soft tissue infections. It typically infects people who already have a lung disease such as cystic fibrosis or someone who is immunocompromised and has recently undergone surgery, tattooing or acupuncture. Other diseases caused by infection from _Mycobacterium_ include tuberculosis and leprosy. Infections from different strains of _Mycobacterium_ have been gradually growing and has been difficult to treat due to its resistance to antibiotics. _Serratia liquefaciens_ was most abundant in the 3H environmental sample with 2075 sequences. The _Serratia_ genus was thought not to be harmful to humans before the 1950s. It wasn't until the United States Navy conducted a secret experiment called "Operation Sea-Spray" wherein which _Serratia marcescens_ and _Bacillus globigii_ bacteria were sprayed over cities in California that include San Francisco, Daly City, Colma, Sausalito, and other parts of the Bay Area from September 20 to 27, 1950. It was enough dosage for the whole Bay Area's population of 800,000 to inhale the bacteria particles. This resulted in pneuomonia, urinary tract infection, and heart valve infection outbreaks as well as the deaths of many people. The U.S. conducted one of the largest human experiments in history without consent from or informing the citizens. The experiment was later explained to be a “vulnerability test” to identify susceptible regions in the event of a biological terrorist attack. San Francisco was chosen for its close proximity to the ocean, as well as its population and tall buildings present throughout its downtown. The city's own natural fog also masked the spray that was administered. A 1951 military report on the experiment summarized the findings and stated, “It was noted that a successful BW [biological warfare] attack on this area can be launched from the sea, and that effective dosages can be produced over relatively large areas”  [@krestonblood]. The military initially reported that _Serratia_ wasn't causing illnesses because infections from the bacteria did not arise. It was a week after the spray was administered that Stanford hospital noted the outbreaks taking place and identified _Serraitia_ as the source of infections and published a report on the outbreaks. In 1981, survivors from the "biological warfare experiment" attempted to sue the government for neglecting the public health of citizens and not obtaining proper consent from the people, however, court ruled that the government was immune from lawsuits. They also mentioned that the US Navy only wanted to see if the bacteria particles would reach the citizens and thus they did not intend to cause disease outbreaks across the city. The lawsuit was appealed all the way to the U.S. Supreme Court, but was still denied. It turns out that this wasn't the last biological warfare test. 239 similar tests were conducted by the U.S. Navy after "Operation Sea-Spray" [@lafreniere10forgiveness].

My BLAST results showed that the top three most abundant organisms from my environmental samples were _Acinetobacter tandoii_, _Salmonella enterica_, and _Listeria monocytogenes_. _Acinetobacter tandoii_ is a Gram-negative aerobic bacterium. It plays an important role in soil for it contributes to mineralization of aromatic compounds. However, it is a source of infection in patients in intensive care units in hospitals. It is resistant to both disinfectants and antibiotics such as penicillin, chloramphenicol, and aminoglycosides allowing it to thrive in hospital settings. Infections from this bacteria seem to take place in a hospital setting immediately after surgery. _Salmonella enterica_ is a pathogenic Gram-negative bacterium that infects the intestinal tract of humans. It is bacteria that is normally found within the intestines of birds and other animals. Humans normally become infected by eating food or drinking water that has been contaminated by feces. Diarrhea and vomiting are signs of infection. Blood tests are conducted to confirm infection. _Salmonella enterica_ is responsible for almost half the bacterial infections in the United States. Infection caused by _Salmonella enterica_ usually goes away on its own after about a week. Some infected people may experience little to no symptoms, others will experience symptoms that include diarrhea, nausea, fever, chills, blood in the stool, and abdomincal cramps. _Listeria monocytogenes_ is a Gram-positive pathogenic bacteria that causes listeriosis, an infection that may cause severe illnesses such as sepsis, meningitis, or encephalitis. It can also cause stillborn and spontaneous abortion in pregnant women. Listeriosis can be transmitted via contaminated food. Infection begins in the intestinal tract and may spread from the intestines to the blood stream or other parts of the body. Fortunately, _Listeria monocytogenes_ is susceptible to antibiotics. After further research, I found that research is currently being done on _Listeria monocytogenes_ as a cancer immunotherapy to treat cervical cancer and has actually entered clinical trials. _Listeria monocytogenes_ is known to infect antigen presenting cells. Advaxix Inc., the company developing the drug, was able to bioengineer the bacterium to be used as a cancer vaccine to be able to deliver tumor antigens directly to the antigen presenting cell thus allowing the tumor to be targeted by the immune system [@singh2011cancer].

I am quite surprised that the top three most abundant organisms from my environmental samples are pathogenic. It has allowed for me to be more thoughtful when cleaning and disinfecting areas such as the sink, bathtub, etc. People spend a lot of time in bathrooms and restrooms on a daily basis so it is important to keep it disinfected to prevent any contact with pathogenic bacteria such as the ones found in my samples from both personal and public restrooms. What surprised me the most is how _Listeria monocytogenes_ is being used for a cancer vaccine that is currently undergoing clinical trials. The illnesses that derive from infection from _Listeria monocytogenes_ are so severe and to think that it can treat women with cervical cancer is mind blowing and shows how far science has come.

Although my BLAST results showed _Acinetobacter tandoii_, _Salmonella enterica_, and _Listeria monocytogenes_ to be the most abundant bacteria overall among my environmental samples, my DADA2-based results show that _Acinetobacter_, _Acidovorax_, and _Pseudomonas_ are the most abundant. As mentioned, _Acinetobacter_ is a pathogenic bacteria that is found to be resistant to both disinfectants and antibitotics. _Acidovorax_ is an organism that was also found in my cultured sample 1P. It is a bacteria known to cause disease in plants. Disease from _Acidovorax_ is indicated by brown blotches on leaves and fruits. _Pseudomonas_ includes bacteria that are pathogenic to animals, plants, and humans. Bacteria that fall under this phyla are resistant to most antibiotics. They have efflux pumps that have the ability to pump out the antibiotics before they are able to act.

_Proteobacteria_ is a phylum of bacteria that covers Gram-negative bacteria. DADA2 showed that his phylum was found in high abundance in both personal and public restrooms. It was interesting to find this in my samples because _Proteobacteria_ includes pathogens such as _Escherichia_ and _Salmonella_. However, free-living bacteria responsible for nitrogen fixation also fall under this phylum. Therefore, I cannot confidently conclude that I am exposed to pathogenic bacteria in both private and public restrooms. _Actinobacteria_ is a phylum of bacteria found in high abundance across my samples. It is a phylum of Gram-positive bacteria that play important roles in soil systems. Bacteria in this phylum are responsible for decomposing organic matter of dead organisms so that the molecules could be taken up by plants. _Firmicutes_ is antoher phylum found in high abundance across my samples per DADA2 results. Contrary to _Proteobacteria_ that contain extremely high GC content, _Firmicutes_, is a phylum of bacteria that is low in GC content. It makes up the largest portion of mouse and human microbiome. _Proteobacteria_, _Proteobacteria_, and _Firmicutes_ are also the most abundant phyla of bacteria found within the human gut. Altogether, the bacteria in the gut play an important role in helping control digestion, the immune system and many other aspects of health. An imbalanced gut microbiota may contribute to diseases such as weight gain, high blood sugar, high cholesterol, and more.

The Shannon Diversity Index (Figure 9) shows that on average, personal restrooms and public restrooms have similar species diversity and evenness. This results adds confirmation to the conclusion that there are no significant differences found between the microbiome of personal bathroom sinks and public restroom sinks. However, the graph shows that personal bathrooms have an overall higher species diversity and evenness compared to public restrooms. The alpha diversity measure of personal bathrooms ranges from approcimately 3.5-4.1 whereas the alpha diversity measure of public restrooms range from approximately 0-3.9. The sample 2P from public restrooms had 4 sequences after trimming, which could explain the wide range in alpha diversity measure that starts at 0 for public restrooms.

Although I was able to find bacteria that play important roles in the human gut, biotechnology manufacturing, and other important roles in soil and crops, my research has shown that the majority of organisms found in both personal bathroom sinks and public restroom sinks are pathogenic and can cause a wide range of diseases in humans. In addition, most of the organisms are known to be resistant to many drugs, antibiotics, and disinfecants. This means that people are coming into close contact with bacteria that can lead to serious disease that may not be easily treated. This means that more reseach needs to conducted on alternative treatments for infections caused by antibitoic resistant bacteria. Healthcare providers must also be aware of the amount of antibiotics being andministered to their patients to prevent populations of antibitic resistant bacteria from growing. The amount of pathogenic bacteria across my samples and sites were similar and all in the thousands range. Therefore, whether someone is using a personal bathroom sink or public restroom sink they will still be in close proximity to all sorts of bacteria including many of which that are becoming more and more antibiotic resistant.

If I were to do further research, there would be some variables to take into consideration that I did not consider prior to starting this experiment. First, I would take a larger number of samples. The six samples I obtained did not provide enough data since the results I achieved showed no signifcant differences across my samples. Other variables I would consider for next time would be things like cleaning patterns of the bathrooms and restrooms (when and how often they are cleaned), the time of day I decide to obtain my samples, and stricter sterility to ensure no contaminants throughout the experiment.

This research has expanded my knowledge of microbial communities in bathroom and restroom sinks. It has given me insight on the types of microorganisms found within surfaces that I and many others come to close contact with on a daily basis. By understanding the genetic structures of bacteria found in everyday environment we can also find ways to understand their mechanisms in infection and causing disease as well as how they have become resistant to disinfectants and even drugs. Previous literature has found that surfaces that have not been cleaned and maintained pose great risk of infectious disease [@reynolds2005occurrence]. This was also portrayed in my experiment seeing that a well kept public restroom such as Apple exhibited much less bacteria than the bathroom in personal homes that were not yet cleaned at the time of field sampling. This means that maintaining a clean environment is necessary to ensure our well being. It is important to know the possible organisms we come into contact with to prepare for potential illness that may arise. My experiment has shown that humans can come into contact with many pathogenic microorganisms. With the increasing number of multi-drug and antibiotic resistant bacteria, we must hurry to find new ways of treating infections that may arise from it. There are papers written on the importance of ownership when it comes to fighting against disease [@tomori2014winning]. By taking ownership of microbial communities found in certain areas, we can then make the commitment to maintain proper hygiene and cleanliness to avoid becoming infected by pathogenic organisms.

# Sources Cited